client/components/Recipe.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { useParams } from 'react-router-dom'
+import { useSelector } from 'react-redux'
+
+export default function Recipe() {
+  const { id } = useParams()
+  const recipes = useSelector((state) => state.recipes)
+  const recipe = recipes[id]
+
+  const label = recipe.recipe.label
+  const largeImage = recipe.recipe.images.LARGE?.url
+  const regularImage = recipe.recipe.images.REGULAR.url
+  const ingredientsArr = recipe.recipe.ingredients
+
+  const dietaryArr = recipe.recipe.healthLabels
+  const selectiveDietary = dietaryArr.filter((word) => {
+    return word === 'Vegan' || word === 'Vegetarian' || word === 'Gluten-Free'
+  })
+
+  return (
+    <div>
+      <h3>{label ? label : 'Hello World'}</h3>
```

Comment:
This can just be
```suggestion
      <h3>{label || 'Hello World'}</h3>
```
But I dont think you want Hello World there

Reply:
Fixed this. Thanks

Reply:
Fixed, thank you